apps/web/src/pages/api/onramp-url-sign/fetch-bsc-connect-availability.ts

```diff
@@ -0,0 +1,76 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import crypto from 'crypto'
+import { z } from 'zod'
+import axios from 'axios'
```

Comment:
Let's use the `fetch` api so we don't need to introduce 3rd party module

Reply:
yeah i was always gonna remove axios when i was finishng this. installed it early on when i firs started this PR. removed now